src/intl/en/page-wallets.json

```diff
@@ -39,12 +39,12 @@
   "page-wallets-seed-phrase-example": "Here's an example:",
   "page-wallets-seed-phrase-snippet": "there aeroplane curve vent formation doge possible product distinct under spirit lamp",
   "page-wallets-seed-phrase-write-down": "Don’t store it on a computer. Write it down and keep it safe.",
-  "page-wallets-slogan": "The key to your digital future",
+  "page-wallets-slogan": "Holding the keys to your digital future",
   "page-wallets-stay-safe": "How to stay safe",
-  "page-wallets-stay-safe-desc": "Wallets require a bit of a different mindset when it comes to safety. Financial freedom and the ability to access and use funds anywhere comes with a bit of responsibility – there’s no customer support in crypto.",
-  "page-wallets-subtitle": "Wallets give access to your funds and Ethereum applications. Only you should have access to your wallet.",
+  "page-wallets-stay-safe-desc": "Financial freedom and the ability to access and use funds anywhere comes with responsibility – there’s no customer support in crypto. You are responsible for keeping your keys safe and secure. If your keys get exposed, you will probably lose your funds.",
+  "page-wallets-subtitle": "Wallets help you access your digital assets and sign in to applications.",
   "page-wallets-take-responsibility": "Take responsibility for your own funds",
-  "page-wallets-take-responsibility-desc": "Centralized exchanges will link your wallet to a username and password that you can recover in a traditional way. Just remember you’re trusting that exchange with custody over your funds. If that company is attacked or folds, your funds are at risk.",
+  "page-wallets-take-responsibility-desc": "Centralized exchanges will link your wallet to a username and password that you can recover in a traditional way. Just remember you’re trusting that exchange with custody over your funds. If that company is turns out to be dishonest or goes bankrupt, your funds are at risk.",
```

Comment:
Perhaps a note about "It is unlikely that an attacker with target an individual, but centralized exchanges act as 'honey-pots' where large amounts of funds are collected, creating a lucrative target for attackers"

Just a thought, feel free to ignore

Reply:
again, agree with the sentiment but think it is good to keep it moving here.

Reply:
The language here feels far too strong for me

Reply:
I don't think we need 'turns out to be dishonest' or any nod to that here tbh. If we look at FTX, their dishonesty wasn't the issue, it was the financial issues that resulted from dishonesty that blew everyone up.